pupa/settings.py

```diff
@@ -9,6 +9,10 @@
 
 # scrape settings
 
+ENABLE_KAFKA = os.environ.get('ENABLE_KAFKA', "False").lower() == "true"
+KAFKA_REPORT_TOPIC = 'post-scrape-reports'
```

Comment:
Topics can be created and fetched programmatically. [kafka-python can do this via `ensure_topic_exists`](https://github.com/mumrah/kafka-python/blob/cd81cf0ec8c1b7e7651374c5d1cbd105d003d352/kafka/client.py#L294). Discovery should straightforward for the forwarder [via `load_metadata_for_topics`](https://github.com/mumrah/kafka-python/blob/cd81cf0ec8c1b7e7651374c5d1cbd105d003d352/kafka/client.py#L310) and filtering based on namespacing. 

For a POC, what's proposed is fine though and shouldn't pose a problem.